R/doeResponseSurfaceMethodology.R

```diff
@@ -671,20 +666,22 @@ doeResponseSurfaceMethodology <- function(jaspResults, dataset, options, ...){
   return()
 }
 
-.responseSurfaceTableAnovaCall <- function(jaspResults, options, rsm, position, dataset) {
+.responseSurfaceTableAnovaCall <- function(jaspResults, options, rsm, i,  position, dataset) {
 
   rsm <- summary(rsm)
 
-  .responseSurfaceAnovaTable(jaspResults, options, dataset, rsm)
+  .responseSurfaceAnovaTable(jaspResults, options, rsm, i, dataset)
 
   return()
 }
 
 
-.responseSurfaceAnovaTable <- function(jaspResults, options, dataset,rsm) {
+.responseSurfaceAnovaTable <- function(jaspResults, options, rsm, i, dataset) {
+
 
+  AnovaTable <- createJaspTable(gettext(paste0("ANOVA for ",
+                                              options[["rsmResponseVariables"]][[i]])))
```

Comment:
So what gettext does is:

1. Put the contents in a separate text file
2. Ask translators to translate this.

Something like `paste0("ANOVA for ", options[["rsmResponseVariables"]][[i]])` cannot be translated (as the translators can never change the position of the index).

Instead, you should do:

```r
gettextf("ANOVA for %s", options[["rsmResponseVariables"]][[i]]) # see ?sprintf
```
There are more instances of this, can you fix them all?